packages/validators/ts-typechecker/src/typecheckValidator.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import path from 'path';
+import findUp from 'find-up';
+import {Validator} from '@parcel/plugin';
+import localRequire from '@parcel/local-require';
+
+import formatDiagnostics from './formatDiagnostics';
+import LanguageServiceHost from './LanguageServiceHost';
+
+export default new Validator({
+  async validate({asset, options}) {
+    let ts = await localRequire('typescript', asset.filePath);
+    let config = {
+      config: await asset.getConfig(['tsconfig.json']),
+      baseDir:
+        path.dirname(
+          await findUp('tsconfig.json', {
```

Comment:
findUp returns undefined if it can't find a tsconfig.

Reply:
TypeScript already handles empty configs, but not sure if we should provide a custom default

Reply:
I also think findUp should just be a temporary solution and there should be some util that's shared with config for resolving config paths.
Fixed the potential issue anyway

Reply:
Well, the tests fail (even with that fix):
If the config above (from `getConfig`) is null, typescript throws an error.

Reply:
ah damn, will update it to default to `{}`

Reply:
Should all be fixed now and no longer even use findUp...